qubesbuilder/executors/qubes.py

```diff
@@ -30,6 +31,24 @@
 log = get_logger("executor:qubes")
 
 
+def quote_list(args: List[Union[str, Path]]) -> str:
+    return " ".join(map(lambda x: quote(str(x)), args))
+
+
+def quote_and_list(cmd: List[List[Union[str, Path]]]) -> str:
+    return " && ".join(map(quote_list, args))
```

Comment:
```suggestion
    return " && ".join(map(quote_list, cmd))
```
But also, maybe name it `cmds`?